src/redux/gas.js

```diff
@@ -258,6 +333,13 @@ export default (state = INITIAL_STATE, action) => {
         selectedGasPrice: action.payload.selectedGasPrice,
         selectedGasPriceOption: action.payload.selectedGasPriceOption,
       };
+    case GAS_UPDATE_CUSTOM_VALUES:
```

Comment:
Looks like this action is never used (along with customGasPrice, customGasPriceEstimate, and customGasPriceFee in the redux state, which are only exposed in the useGas hook)